From 663ea6df17d7ac477ca20c2b2712c311006e1103 Mon Sep 17 00:00:00 2001
From: Christian Heinz <christian.ch.heinz@gmail.com>
Date: Fri, 26 Nov 2010 03:21:17 +0100
Subject: [PATCH] fix installation of `modules.builtin.bin'

`modules.builtin.bin' is installed like a regular file, thereby ending
up in the wrong place when `--kmoddir' is in effect. Fix this by
specifying the installation destination.
---
 modules.d/90kernel-modules/install | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90kernel-modules/install b/modules.d/90kernel-modules/install
index 05819f5..90de1c0 100755
--- a/modules.d/90kernel-modules/install
+++ b/modules.d/90kernel-modules/install
@@ -2,4 +2,4 @@
 [ -f /etc/modprobe.conf ] && dracut_install /etc/modprobe.conf
 dracut_install $(find /etc/modprobe.d/ -type f -name '*.conf')
 inst_hook cmdline 01 "$moddir/parse-kernel.sh"
-inst "$srcmods/modules.builtin.bin"
+inst "$srcmods/modules.builtin.bin" "/lib/modules/$kernel/modules.builtin.bin"
